charts/stable/nullserv/values.yaml

```
@@ -40,6 +40,9 @@ service:
       main:
         port: 80
         targetPort: 80
+  https:
+    type: NodePort
+    ports:
```

Please don't set type nor use nodePort
```suggestion
  https:
    enabled: true
    ports:
```